app/filters.js

```diff
@@ -57,16 +56,16 @@ module.exports = (env) => {
   filters.govukDateAtTime = date => {
     const govukDate = filters.govukDate(date)
     const time = filters.time(date)
-    return govukDate + " at " + time
+    return govukDate + " at " + (time).toLowerCase()
```

Comment:
is the time filter not lowercase in the first place? Can it be?

Reply:
The default case for Luxon is upper. I'll move this to the time filter for consistency across all usage.